ambariclient/base.py

```diff
@@ -396,7 +397,11 @@ class Model(object):
     min_version = OLDEST_SUPPORTED_VERSION
 
     def __init__(self, parent, data=None):
-        self._data = data if data is not None else {}
+        if data is None:
+            data = {}
+
+        self._data = dict((key, value) for key, value in six.iteritems(data)
```

Comment:
The issue is that `QueryableModelCollection.create` passes all keyword arguments to both the `model.__init__` (as `data`) and to `model.create`.  I was concerned that sending the `validate` keyword to the constructor in `data` would cause problems, since it's not a part of the model.